nirc_ehr/src/org/labkey/nirc_ehr/dataentry/section/NIRCTissueDispositionFormSection.java

```diff
@@ -0,0 +1,42 @@
+package org.labkey.nirc_ehr.dataentry.section;
+
+import org.labkey.api.view.template.ClientDependency;
+
+import java.util.List;
+
+public class NIRCTissueDispositionFormSection extends BaseFormSection
```

Comment:
Also need a trigger for this one `require("ehr/triggers").initScript(this);`

Reply:
done